install/kubernetes/quick-hubble-install.yaml

```diff
@@ -221,7 +221,7 @@ roleRef:
   name: hubble-generate-certs
 subjects:
 - kind: ServiceAccount
-  name: hubble-generate-certs
+  name: hubble-generate-certs 
```

Comment:
Where is the trailing space coming from? :thinking:

Reply:
Fixed.